cram_3d_world/cram_urdf_projection_reasoning/src/check-collisions.lisp

```diff
@@ -170,7 +186,7 @@ Store found pose into designator or throw error if good pose not found."
                           (cut:equalize-two-list-lengths left-poses right-poses)
                         (dotimes (i (length left-poses))
                           (urdf-proj::move-tcp (nth i left-poses) (nth i right-poses)
-                                               :allow-all)
+                                               :allow-al nil nil nil move-base)
```

Comment:
meh, you ate the second L. how did you even test this, even with your own designators...